pkg/tokenutil/grpc.go

```diff
@@ -43,14 +43,34 @@ func (t ContextToken) GetRequestMetadata(ctx context.Context, uri ...string) (ma
 	}
 
 	return map[string]string{
+		// TODO remove passing the token in authorization after the next major version.
 		"authorization": finalT,
+		MetadataKey:     finalT,
```

Comment:
Do you know this might push us close to a size limit? I did some quick looking, and it looks grpc metadata winds up in http/2 headers, which are compressed but also maybe have a 4k byte limit for most load balancers?

Reply:
because they're compressed, the second value is basically "free"

Reply:
AH good point. And I don't think this will show up in the headers the UI sends either.